### DISCUSSION TITLE
Digital Theater Production Company: PLACEHOLDER FOR NEED

### DISCUSSION THREAD

**Discussion by ybressler** [pair-cg/general-contributions#64] [User Stories]
# 1. Need:
_What is your need?_

Ivanka is the President of the Macedonian Center ITI/PRODUKCIJA, an NPO dedicated to producing digital theatrical productions. Ivanka wants to.... _provide some metrics to recieve government grants?_

I need to write an effective grant and for that I need access to the right information so I know how to explain and position the work that I do for the grant organization that I am appealing to.

# 2. Action:
_What is an action that can be taken towards fulfilling that need?_

1. I need to write an effective grant by this specific program _______________________, which requires these specific measurables...

# 3. Desired Outcome:
_What is the desired outcome?_

I get money from the granting body.

# 4. Perspective:
_From what perspective is this user story coming from?_

# 5. Stakeholder Group:
_Which stakeholder group most closely fits the perspective of this User Story?_

# 6. Additional info:
_Here's more information._

**Comment by ybressler** [comment]
**Who this person is:**
* President and Head of Macedonian Center ITI/PRODUKCIJA
* Executive

**Reply by ybressler** [reply]
Also creating video theater productions

**Comment by ybressler** [comment]
When's the last time you triwed to do ______ but couldn't because _____.

**Reply by ybressler** [reply]
Video theater --> 3 production:
* 2019
* 2020

Desire to get more funding.

**Reply by ybressler** [reply]
Based in Macadonia, complicated when analyzing projects. How do you determine impact of a video performance?

**Comment by ybressler** [comment]
**Some follow up Q's to Ivanka:**
* Grant analysts will be analyzing your digital productions to determine if they should receive a grant.
  * What are your data needs for this to be a "good experience" ?
  * _A possible answer is --> Grant analysts can determine the cultural impact / number of people who saw this production and determine grant eligibility._
* How do grant analysts determine eligibility for "in-person" theatrical productions?
* How do grant analysts determine eligibility for other grant applications?

**Reply by ybressler** [reply]
**_What are your goals?_** 

Get more money from private organizations.
* What are the needs of the organizations giving out the money?
* Can you align your grant to these organizations?

**Comment by mariel-marshall** [comment]
Proposed Need: I need to write an effective grant and for that I need access to the right information so I know how to explain and position the work that I do for the grant organization that I am appealing to.
